src/scripts/components/VideoOverlay.js

```diff
@@ -30,17 +30,28 @@ const Overlay = styled.div`
 
 class VideoOverlay extends Component<Props, void> {
   render () {
```

Comment:
FYI on my branch I'm re-working this component to match the designs so whatever you have here will likely change